### PR TITLE
[bitnami/etcd] add etcd nodeselector & toleration on disasterRecovery

### LINKED ISSUE #7769: [bitnami/etcd] - include nodeSelector for snapshotter job
**Which chart**:
bitnami/etcd

**Is your feature request related to a problem? Please describe.**
I'm using EFS NFS (aws-efs-csi-driver) in AWS EKS. It spawns one pod per node to hook up the nfs. 
etcd snapshotter job https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/cronjob.yaml has no ability to define the nodeSelector for the pod, which would be nice to have.

If enabled, it's possible to use http aws-efs-csi-driver's nodeSelector and reduce number of pods spawned. 

**Describe the solution you'd like**

add nodeSelector like this

```
disasterRecovery:
  nodeSelector:
      etcd: "true"
```

**Describe alternatives you've considered**

Spawn a lot more pods aws-efs-csi-driver (one per Node, however that EFS is only used by etcd in the cluster)

### PR DESCRIPTION
**Pull request by cyril-corbon**
**Description of the change**
add nodeSelector & tolerations on disasterRecovery cronJob

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes  #7769

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/etcd/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 3.5.1
+appVersion: 3.6.1
```
Comment (miguelaeh) [inline bitnami/etcd/Chart.yaml:4]: Please don't change the `appVersion`

File: bitnami/etcd/README.md [hunk 2]
```diff
@@ -262,6 +262,8 @@ The command removes all the Kubernetes components associated with the chart and
 | `disasterRecovery.cronjob.podAnnotations`       | Pod annotations for cronjob pods                                        | `{}`           |
 | `disasterRecovery.cronjob.resources.limits`     | Cronjob container resource limits                                       | `{}`           |
 | `disasterRecovery.cronjob.resources.requests`   | Cronjob container resource requests                                     | `{}`           |
+| `disasterRecovery.cronjob.nodeSelector`   | Cronjob container node selector                                    | `{}`           |
+| `disasterRecovery.cronjob.tolerations`                           | Tolerations for cronjob assignment                                                            | `[]`            |
```
Comment (miguelaeh) [inline bitnami/etcd/README.md:266]: Please use the [Readme Generator For Helm](https://github.com/bitnami-labs/readme-generator-for-helm) to generate this in the proper format

File: bitnami/etcd/values.yaml [hunk 3]
```diff
@@ -728,6 +728,16 @@ disasterRecovery:
       ##    memory: 1Gi
       limits: {}
       requests: {}
+
+    ## @param disasterRecovery.cronjob.nodeSelector [object] Node labels for pod assignment
+    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+    ##
+    nodeSelector: {}
+    ## @param disasterRecovery.cronjob.tolerations [array] Tolerations for pod assignment
```
Comment (miguelaeh) [inline bitnami/etcd/values.yaml:736]:
```suggestion
    ## @param disasterRecovery.cronjob.tolerations Tolerations for pod assignment
```

File: bitnami/etcd/values.yaml [hunk 4]
```diff
@@ -728,6 +728,16 @@ disasterRecovery:
       ##    memory: 1Gi
       limits: {}
       requests: {}
+
+    ## @param disasterRecovery.cronjob.nodeSelector [object] Node labels for pod assignment
```
Comment (miguelaeh) [inline bitnami/etcd/values.yaml:732]:
```suggestion
    ## @param disasterRecovery.cronjob.nodeSelector Node labels for pod assignment
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!
Please take a look at my comments

**Comment by cyril-corbon** [comment]
review applied & lint done

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**